Packs/CrowdStrikeFalcon/TestPlaybooks/playbook-CrowdStrikeFalcon-Test.yml

```diff
@@ -2624,8 +2684,43 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 1042.5,
-          "y": 9265
+          "x": 275,
+          "y": 9440
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "68":
+    id: "68"
+    taskid: b96cbefb-2592-4827-832a-c22c6e9d7b0f
+    type: regular
+    task:
+      id: b96cbefb-2592-4827-832a-c22c6e9d7b0f
+      version: -1
+      name: Get device login history
+      script: '|||cs-falcon-get-device-login-history'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      device_id:
+        simple: c1a4575cd124419db5ed3c1e8ddc2c04
```

Comment:
you should add some method to get this id from the integration (so this playbook will work in every environment, not just in yours).